packages/components/src/components/button/button.jsx

```diff
@@ -20,6 +20,7 @@ const Button = ({
     is_loading,
     is_submit_success,
     is_button_toggle,
+    is_circle,
```

Comment:
I think it's better to name this as `is_icon_circle` since the prop is related to icon

Reply:
@balakrishna-binary What I mean is `if the button is circle then choose that icon`, so I believe this name is ok, what is your idea?

Reply:
@balakrishna-binary What I mean is, `if the button is circle use this icon`, so I believe this name is ok, what do you think?

Reply:
alright then 👍